OneCommunityGlobal/HighestGoodNetworkApp
# Luis - Hours logged should be red

# Description

<img width="779" alt="Screenshot 2023-09-26 at 8 01 08 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/8df32e5f-f41c-4860-896e-4d77eb6177bc">
In short if hours haven't been fulfilled the text should be color red. Just like the "not provided" text



## Main changes explained:
Added a css class to weeklysummariesreport.css called hoursNotFulfilledwith color red.
If the hours aren't fulfilled the styling is added


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user or owner
4. go to dashboard→ reports → weekly reports
5. All user's who haven't fulfilled their hours should be red.

## Screenshots or videos of changes:
<img width="1621" alt="Screenshot 2023-09-26 at 8 13 31 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/7607ca8f-af3f-4fcf-b7e4-e4da78ee340e">

## Note:
There is one user who's text doesn't become red. Because it has default styling. I am unsure why it has default styling since, I adjusted the jsx. 

<img width="1438" alt="Screenshot 2023-09-26 at 8 13 11 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/c22bb507-2c12-4d4f-946a-9fb0149ffc3f">
